<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1028</width>
    <height>571</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1028</width>
    <height>571</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1550</width>
    <height>1000</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>1370</width>
    <height>1030</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="pushButtonP1">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>360</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>频率测量</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonP2">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>360</y>
     <width>171</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>数码管静态显示测试</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textCommd">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>961</width>
     <height>331</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonClear">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>清空</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonOPEN">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>410</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>打开串口</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxCOM">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxBand">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>360</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>9600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>57600</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>115200</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>410</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
